src/cmd/ksh93/features/nvapi.h

```diff
@@ -0,0 +1,32 @@
+/* : : generated from nvapi by iffe version 2013-11-14 : : */
+#ifndef _NV_API_H
+#define _NV_API_H	1
+#define _sys_types	1	/* #include <sys/types.h> ok */
+#ifndef _API_nv
+#define _API_nv	_API_shell
+#endif
+#define NVAPI(rel)	( _BLD_nv || !_API_nv || _API_nv >= rel )
+
+#ifndef _NV_API_IMPLEMENT
+#define _NV_API_IMPLEMENT	1
+
+
+#if !defined(_API_nv) && defined(_API_DEFAULT)
```

Comment:
This line is a good example why we need to rewrite this from scratch at some point. Notice that this can never be true because six lines earlier `_API_nv` is defined. I'm not asking that this be fixed now as it's better if this remains the literal output of iffe in this first step towards a simpler, lighter weight, mechanism.